### PR TITLE
go@1.12 1.12.13

### PR DESCRIPTION
**Pull request by chenrui333**
bump the version

### CODE REVIEW

File: Formula/go@1.12.rb [hunk 1]
```diff
@@ -13,8 +13,6 @@ class GoAT112 < Formula
 
   keg_only :versioned_formula
 
-  depends_on :macos => :yosemite
```
Comment (leonklingele) [inline Formula/go@1.12.rb]: Rather change this to `depends_on :macos => :el_capitan`

Comment (chenrui333) [reply Formula/go@1.12.rb]: That is right, let me do that.

Comment (leonklingele) [reply Formula/go@1.12.rb]: Wait — I got confused. The Go 1.12 release line _does_ still support Yosemite, so don't change the line at all :)

Comment (chenrui333) [reply Formula/go@1.12.rb]: yeah, I just realized that too, I guess it only applies to 1.13. 👻

Comment (leonklingele) [reply Formula/go@1.12.rb]: I'm sorry! 😳

**Review by AlekSi — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**